### ISSUE TITLE
Pull requests

### ISSUE THREAD

**Issue by akash-vartak** [comandosgit/comandosgit.github.io#22]
Can I open pull requests in English? I see that your website is not in English so I thought I should ask you beforehand.

**Comment by rafaelstz** [comment]
yes sure, PR can be in English, I'm deciding to leave in Portuguese yet, if I leave in English then translate later.
Thanks
